terraform/api.tf

```diff
@@ -44,6 +44,8 @@ module "api" {
     DJANGO_DANDI_JUPYTERHUB_URL                    = "https://hub.dandiarchive.org/"
     DJANGO_DANDI_DEV_EMAIL                         = var.dev_email
     DJANGO_DANDI_ADMIN_EMAIL                       = "info@dandiarchive.org"
+    DANDI_INSTANCE_NAME                            = "DANDI"
+    DANDI_DATACITE_DOI_ID                          = "48324"
```

Comment:
as decided to go for prefix, needs to be adjusted

Reply:
done